.gitignore

```diff
@@ -1,5 +1,5 @@
 # Created by https://www.gitignore.io/api/osx,linux,node,vim
```

Comment:
Looks good!  You may also consider adding *.log to avoid uploading things like npm-debug.log as well, not an issue in this case though :)